### PR TITLE
Adding default for users

### PR DESCRIPTION
**Pull request by hub-cap**

### CODE REVIEW

File: reddwarf/instance/service.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ def create(self, req, body, tenant_id):
         flavor_ref = body['instance']['flavorRef']
         flavor_id = utils.get_id_from_href(flavor_ref)
         databases = populate_databases(body['instance'].get('databases'))
```
Comment (cp16net) [inline reddwarf/instance/service.py]: What about databases?

Comment (hub-cap) [inline reddwarf/instance/service.py]: ya thatll prolly fail too, ill update

Comment (hub-cap) [inline reddwarf/instance/service.py]: fixed

**Comment by cp16net** [comment]
+1

**Comment by rnirmal** [comment]
+1

**Pull request merged**